about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
+
+### Citation management and bibliometrics:
+
+Tools that facilitate managing references, such as for writing manuscripts,
+creating CVs or otherwise attributing scientific contributions, or accessing,
+manipulating or otherwise working with bibliometric data. (Example: [Example from rOpenSci - RefManageR](https://github.com/ropensci/software-review/issues/119))
+
+### Data visualization & analysis
+These are packages that enhance a scientists experience visualizing and
+analyzing data.
+
+* Examples: [PyGmt - (also spatial and data munging)](https://github.com/pyOpenSci/software-submission/issues/43),
+
+### Database software bindings
+
+ Bindings and wrappers for database APIs.
+
+ * Example: [Example from rOpenSci - rrlite](https://github.com/ropensci/software-review/issues/6)
+
+
+## Specialty topics
+
+In addition, we have some specialty topics with a broad domain focused scope.
+
+### Geospatial
+
+Packages focused on the retrieval, manipulation, and analysis of spatial data.
+
+* Examples: [PyGmt](https://github.com/pyOpenSci/software-submission/issues/43),
+[Moving Pandas ](https://github.com/pyOpenSci/software-submission/issues/18)
+
+### Pangeo
+
+We have a [partnership with Pangeo](../partners/pangeo). Often times packages submitted as a part of that partnership are also in the geospatial domain.
+
+* Examples: [xclim - under review now](https://github.com/pyOpenSci/software-submission/issues/73)
+
+### Education
+
+Packages to aid with instruction.
+<!--TODO -  Earthpy fit in this category but it also needs to be sunsetted -->
+
+## Package technical scope
+
+To be in technical scope for a pyOpenSci review, your package:
+
+* Should have maintenance workflows documented.
+* Should be structured in a way that someone else could contribute to it.
+* Should declare vendor dependencies using standard approaches rather than including code from other packages within your repository.
+
+### Notes on scope categories
+- pyOpenSci is still developing as a community. If your scientific Python
+package does not fit into one of the categories or if you have any other
+questions, we'd encourage you to open a pre-submission inquiry. We're happy to help.
+- Data visualization packages come in many varieties, ranging from small
+hyper-specific methods for one type of data to general, do-it-all packages
+(e.g. matplotlib). pyOpenSci accepts packages that are somewhere in between the
+two. If you're interested in submitting your data visualization package, please
+open a pre-submission inquiry first.
+
+## Python package technical scope
+
+pyOpenSci may continue to update its technical scope criteria for package
+review as more packages with varying structural approaches are reviewed.
+Your package **may not be in technical scope** for us to review at this time if
+fits any of the out-of-technical-scope criteria listed below.
```

Comment:
For me this is a bit confusing: "your package may not be in technical scope if it fits any of the out-of-technical-scope"
(nit: it's also kind of tautological and repetitive)

Should we have some criteria for what **is** in scope? And then contrast those with the examples below?

E.g.:
- in scope
  - pure Pyton
  - Python with built extensions
  - Python package that can be built with conda package from community channels such as conda-forge, bioconda, etc.
- out of scope: exceedingly complex in terms of structure or maintenance needs
  - this is difficult to define but we provide two examples

Reply:
i like this. I've been thinking about this because in some ways this part of our policy is very specific to  a single package that we looked at. The generalized statement might be something like... 

the package needs to be setup in a way that another maintainer can (Easily?) step in and take over. ... you know something to that effect? because that is core to our mission. so the complexity needs to be well documented, justified and implementable by someone new. What do you think about some combination of that message PLUS what you wrote above which i like?

Reply:
Yes, I think it makes sense to (a) preface with your language about what one of the goals of a good development workflow is --it's easy to take over maintenance-- then (b) restate concretely as I did


also possibly link to the schematic you're developing for packaging tools ("for example, if you're unsure about packaging workflows see this guide")

Reply:
totally... ok i've updated. i'll add graphics i think later - that one is still in progress but i think it's a great idea!! we just need to get that packaging info online first :)